loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/tools/RunLoadFlowTool.java

```diff
@@ -162,15 +158,14 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
         if (network == null) {
             throw new PowsyblException("Case '" + caseFile + "' not found");
         }
-        LoadFlow loadFlow = defaultConfig.newFactoryImpl(LoadFlowFactory.class).create(network, context.getShortTimeExecutionComputationManager(), 0);
 
         LoadFlowParameters params = LoadFlowParameters.load();
         if (line.hasOption(PARAMETERS_FILE)) {
             Path parametersFile = context.getFileSystem().getPath(line.getOptionValue(PARAMETERS_FILE));
             JsonLoadFlowParameters.update(params, parametersFile);
         }
 
-        LoadFlowResult result = loadFlow.run(network.getVariantManager().getWorkingVariantId(), params).join();
+        LoadFlowResult result = LoadFlow.findDefault().run(network, context.getShortTimeExecutionComputationManager(), params);
```

Comment:
Is it a good idea to create a static method to replace LoadFlow.findDefault().run(....) by:
```
LoadFlow.run(....);
```
that will find the default implementation.

Reply:
That would be nice but it would collide with instance methods. It is not possible in Java to have this kind of overload
```java
void a();
static void a();
```

Reply:
We'll try to find a more convenient way to solve this.